torch/csrc/jit/operator_upgraders/utils.cpp

```diff
@@ -49,5 +49,27 @@ bool isOpSymbolCurrent(const std::string& name, size_t current_version) {
   return true;
 }
 
+std::vector<std::string> loadPossibleHistoricOps(
+    const std::string& name,
+    c10::optional<size_t> version) {
+  std::vector<std::string> possibleSchemas;
+
+  if (!version.has_value()) {
+    return possibleSchemas;
+  }
+
+  for (const auto& entry : get_operator_version_map()) {
+    auto old_symbol_name = entry.first;
+    if (old_symbol_name.find(name) == 0) {
```

Comment:
`old_symbol_name` is a string I think, and `name` should be an exact match for it, why using `find` which checks if they are substring relationships? Is it to get rid of overload name? If so, we should have more strict logic that separates `old_symbolc_name` into two parts by `.`. Current implementation could be confused by symbol names that prefix each other.

Reply:
This is because name is actually a symbol so it doesn't have the overload in it. schema_matching is responsible for finding all the overloads for this name and do schema matching to find the correct overload. But, operator_version_map stores the schema name with its' overload, so we need to check if the old_symbol_name is a valid overload for name. That being said, you are right that the check needs to be stricter.